$NetBSD: patch-ad,v 1.3.2.2 2011/02/05 10:12:04 spz Exp $

Fix a hang when trying to COPY to a non existent mailbox via IMAP.
Patch taken from here:

http://hg.dovecot.org/dovecot-2.0/rev/826981b2c5c4

--- src/imap/cmd-copy.c.orig	2010-03-25 15:03:33.000000000 +0000
+++ src/imap/cmd-copy.c	2011-02-02 22:10:21.000000000 +0000
@@ -126,7 +126,7 @@
 	case MAILBOX_NAME_NOINFERIORS:
 		client_fail_mailbox_name_status(cmd, mailbox,
 						"TRYCREATE", status);
-		return NULL;
+		return TRUE;
 	}
 
 	if (mailbox_equals(client->mailbox, dest_ns, storage_name))
